go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.4
 	github.com/ti-mo/conntrack v0.5.0
-	github.com/vishvananda/netlink v1.1.1-0.20211101163509-b10eb8fe5cf6
+	github.com/vishvananda/netlink v1.2.1-beta.2
```

Comment:
gotta love how the beta release dates back to June 2022, with no release since...

Reply:
Yes... the latest version is still v1.21-beta.2.